lib/matplotlib/path.py

```diff
@@ -528,22 +587,39 @@ def contains_path(self, path, transform=None):
             transform = transform.frozen()
         return _path.path_in_path(self, None, path, transform)
 
-    def get_extents(self, transform=None):
+    def get_extents(self, transform=None, **kwargs):
         """
-        Return the extents (*xmin*, *ymin*, *xmax*, *ymax*) of the path.
+        Get size of Bbox of curve (instead of Bbox of control points).
```

Comment:
Since there is no longer an approximate/exact split, the "instead of" parenthetical is not needed any more, I think?